src/main/app/index.js

```diff
@@ -155,21 +154,15 @@ class App {
         () => {
           const preferences = this._accessor.preferences
           const { theme } = preferences.getAll()
-          let setedTheme = null
-          if (systemPreferences.isDarkMode() && theme !== 'dark') {
+
+          // Application menu is automatically updated via preference manager.
+          if (systemPreferences.isDarkMode() && theme !== 'dark' &&
```

Comment:
@Jocs Please test and verify this change.

Reply:
ok